docs/source/API/core/profiling/profiling_section.rst

```diff
@@ -0,0 +1,58 @@
+``Profiling::ProfilingSection``
+===============================
+
+.. role:: cppkokkos(code)
+   :language: cppkokkos
+
+Defined in header ``<Kokkos_Profiling_ProfileSection.hpp>``
```

Comment:
Hadn't even realized the name is "ProfileSection" and not "ProfilingSection"...
Yuk